swarm/storage/mru/resource_test.go

```diff
@@ -464,7 +464,7 @@ func TestMultihash(t *testing.T) {
 		OwnerValidator: rh.ownerValidator,
 	}
 	// test with signed data
-	rh.chunkStore.Close()
+	// rh.chunkStore.Close()
```

Comment:
I don't know if you really have to remove this or not, but if it has to be removed let's delete the line, and not just comment it out

Reply:
Yes, this lines are really not needed even as comments as `rh.Close` does exactly that.

Reply:
with #700 merged this file is untouched